apps/love-producer/values-tucson-teststand.yaml

```diff
@@ -72,6 +72,9 @@ producers:
   camerahexapod:
     name_index: MTHexapod:1 --log-level 10
     WEBSOCKET_HOST: love-nginx-service/love/manager/producers/ws/subscription
+  cbp:
```

Comment:
What happened to the linearstage?

Reply:
The hardware was shipped and so no longer existed in Tucson's calibration lab so I removed them from TTS but I forgot to add them back since they will use the simulation mode.